### ISSUE TITLE
Softmax runtime warning

### ISSUE THREAD

**Issue by hiramustafa77** [svishwa/crowdcount-cascaded-mtl#22]
UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
this is the original code line: density_cls_prob = F.softmax(density_cls_score) 
So how do I know whether dim should be taken '0' or '1'
